lib/metrics.js

```diff
@@ -17,9 +17,6 @@ const metricsIds = {
 
 module.exports = {
   hiddenMetrics: [
-    metricsIds.FV,
-    metricsIds.VC100,
-    metricsIds.VC85,
```

Comment:
We should have them. Because we don't wanna show them on chart but show in file output.